src/FormBuilder/resources/js/form-builder/src/styles/_components.scss

```diff
@@ -297,3 +297,13 @@ This creates a consistent separator between the tabs and the rest of the sidebar
     }
 }
 
+.give-error-modal {
+    ul {
+        list-style: disc;
+        padding-inline-start: 1rem;
+    }
+
+    &__dismiss-button {
+        float: right;
+    }
+}
```

Comment:
Whenever possible, i'd like to move away from dumping styles in here (especially components) and start co-locating styles with their component by importing directly.  The final bundle goes to the same place but is much easier to maintain in practice.

Reply:
Agreed. I actually tried to use a CSS module on a previous PR and the build system threw a complete fit. It seemed to have an issue with both processing the SCSS this way _and_ having CSS modules.

Reply:
Resolved in c68a84d